optimum_benchmark/benchmarks/inference/benchmark.py

```diff
@@ -77,16 +79,24 @@ def run_forward_tracking(self, backend: "Backend") -> None:
         if self.config.memory:
             LOGGER.info("\t+ Tracking forward pass peak memory")
             memory_tracker = memory_tracker_class_for_backend[backend.config.name](backend)
-            with memory_tracker.track(interval=self.config.duration // 100):
+            with memory_tracker.track(interval=self.forward_latency / 10):
                 _ = backend.forward(forward_input)
 
             self.forward_peak_memory = memory_tracker.get_peak_memory()
             LOGGER.info(f"\t+ Forward pass peak memory: {self.forward_peak_memory} (MB)")
 
+        if self.config.emissions:
+            LOGGER.info("\t+ Tracking forward pass emissions")
+            emissions_tracker = EmissionsTracker()
+            with emissions_tracker.track(interval=self.forward_latency / 10, file_prefix="forward"):
+                _ = backend.forward(forward_input)
+
+            self.forward_emissions = extract_three_significant_digits(emissions_tracker.get_emissions())
+            LOGGER.info(f"\t+ Forward pass emissions: {self.forward_emissions} (kgCO2)")
+            LOGGER.info("\t+ A detailed report is available at the benchmark directory")
```

Comment:
If the benchmark directory is accessible from here, I would add it to the log with a f-string

Reply:
done